debian/clearwater-snmpd.postinst

```diff
@@ -58,6 +58,7 @@ case "$1" in
         cp /etc/snmp/snmpd.conf.clearwater-snmpd /etc/snmp/snmpd.conf
 
         # Create the folder to store the statistics UNIX domain sockets.
+        rm -rf /var/run/clearwater
```

Comment:
This is really destructive. What if some other package wants to use it? When `clearwater-snmpd` gets installed the other package's data will just get deleted. `clearwater-snmpd` does not own this directory, so really shouldn't delete it. Why does it need to anyway?